static/js/components/PersonalTab.js

```diff
@@ -25,19 +26,27 @@ class PersonalTab extends React.Component {
   updateGender(gender) {
     const { dispatch, profile: { profileCopy } } = this.props;
```

Comment:
was just thinking about this a little more.

we currently create `profileCopy` on app initialization - not when we actually start editing.

if we rename it to something relating to edits in progress, I think it should only be created when the user actually starts editing stuff (maybe when the first `onChange` callback fires?).